com.github.Bleuzen.FFaudioConverter.json

```diff
@@ -0,0 +1,74 @@
+{
+    "app-id": "com.github.Bleuzen.FFaudioConverter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "ffaudioconverter",
+    "rename-desktop-file": "ffaudioconverter.desktop",
+    "rename-appdata-file": "FFaudioConverter.appdata.xml",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name": "libmp3lame",
+            "config-opts": ["--enable-shared",
+                            "--enable-nasm",
+                            "--prefix=/app",
```

Comment:
Not needed.

Reply:
Sure? I can't compile ffmpeg without it.

Reply:
I meant `--prefix=/app`

Reply:
You might need to add `cleanup` to this module too and clean uneeded header files or docs or whatever stuff that are not used/necessery

Reply:
Ok, I removed it.

Reply:
cleanup is now also done